backend/vehicle/views.py

```diff
@@ -76,3 +66,27 @@ def get(self, request, vehicle_id, *args, **kwargs):
             # return Response(serialized_data.data, status=status.HTTP_200_OK)
         except vehicle.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+class VehicleFilterList(APIView):
+    """
+    Get list of vehicles based off of filter
+    Takes limit + offset from url
+    """
+
+    def get_queryset(self):
+        queryset = infinite_filter(self.request)
+        return queryset
+
+    def get(self, request):
+        url_parameter = request.GET.get("p")
```

Comment:
@kevinrczhang Could we use `request.data.get("p")` instead to be similar to the other code.

Reply:
since p is the search term, i think it's better if we put it in request.GET just because this technically isn't data that we're passing in - its parameters